### ISSUE TITLE
/_layouts/15/userphoto.aspx to get user photo

### ISSUE THREAD

**Issue by PooLP** [SharePoint/sp-dev-docs#7392]
### What type of issue is this?

Question

### What SharePoint development model, framework, SDK or API is this about?

not applicable

### Target SharePoint environment

SharePoint Online

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [ ] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [X] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Issue description

Hi,
I'm use this url to retrieve users avatar (replace "accountName" by email) in SPFx dev :
/_layouts/15/userphoto.aspx?size=M&accountname=accountname

I never had a problem, but since this morning, it does not work on of my all tenants. Are you also having the problem?

This method is also used by @hugoabernier in are livePersonaCard ;)

**Comment by bcameron1231** [comment]
I've noticed some weird behaviors recently where SharePoint Photo Sync has been replacing user's photos with empty placeholders. 

If you go to the user photos library in mysites... are your user photos blank like mine?

This is definitely a regression that I saw, which started about a week and a half ago.

**Comment by PooLP** [comment]
The users photos in 0365bar appear blank in personal sites, but i don't find a user photo library on my tenant.

The users photos reappearing this morning.

I have a little suggestions :

If the user's photo does not exist, there should not be the returned replacement photos (the silhouette), because if we use the Persona Fluent UI, it is this replacement photo that appears instead of the initials.

**Comment by PooLP** [comment]
After one month+, the problem still exists. It's aleatory.

**Comment by bcameron1231** [comment]
@PooLP Yup. Unfortunately, User Photos have always been an issue in SPO. I would submit a support request and see if you can get any help there. I ended up building my own user profile synchronization tool and use photos stored on Azure CDN instead because SharePoint just wasn't reliable enough. 👍

**Comment by PooLP** [comment]
In the meantime, i'm use graph to get the user photo.

The attitude of MS support when reporting bugs is unacceptable, that's why I'm posting some bugs here, it's the desperation that makes me do this :D
